TimedArithmeticQuiz/src/App.jsx

```diff
@@ -0,0 +1,66 @@
+import { useCallback, useEffect, useRef, useState } from "react"
+import { QuizHeader } from "./components/QuizHeader"
+import { quizConfig } from "./data/quizConfig"
+import { Question } from "./components/Question";
+import { Quiz } from "./models/Quiz";
+import { QuestionHeader } from "./components/QuestionHeader";
+import { QuizComplete } from "./components/QuizComplete";
+
+
+function App() {
+  const [score, setScore] = useState(0);
+  const [quiz,] = useState(new Quiz(quizConfig.numberRange, quizConfig.operatrs ,quizConfig.totalQuestionCount));
```

Comment:
I think the convention is to define a setter with `_` to show that it's not being used? I might be confusing it with my Rails conventions tho.

@shivamvinsol ?